### ISSUE TITLE
Your training set is empty. Your validation set is empty

### ISSUE THREAD

**Issue by miltosc** [jantic/DeOldify#228]
Hello!

while trying to train with ColorizeTrainingVideo.ipynb at the command 
```
data_gen = get_data(bs=bs, sz=sz, keep_pct=keep_pct)
```

it throws the following warning although i have folders train, val, test inside CLS-LOC folder

```
/lib/python3.6/site-packages/fastai/data_block.py:442: UserWarning: Your training set is empty. If this is by design, pass `ignore_empty=True` to remove this warning.
  warn("Your training set is empty. If this is by design, pass `ignore_empty=True` to remove this warning.")
/lib/python3.6/site-packages/fastai/data_block.py:445: UserWarning: Your validation set is empty. If this is by design, use `split_none()`
                 or pass `ignore_empty=True` when labelling to remove this warning.
  or pass `ignore_empty=True` when labelling to remove this warning.""")
```

Please help.
Thank you.

**Comment by jantic** [comment]
It turns out I omitted the step from ColorizeTrainingStable that generates the black and white training images from ImageNet:

![image](https://user-images.githubusercontent.com/179759/81612442-abcb7480-9391-11ea-9aa6-fba05a10e4bf.png)

I think the intention I had was that you would run the ColorizeTrainingStable.ipynb notebook first, as it says this on top of the video training notebook:

> It's assumed that there's a pretrained generator from the ColorizeTrainingStable notebook available at the specified path.

So if you'd had run ColorizeTrainingStable first you would already have the b&w images generated.  If not, that's probably your problem here. Anyway, I'll go ahead and add that step too ColorizeVideoTraining notebook to allow for the possibility that you'd want to jump straight to video training with the pretrained stable model I've uploaded already. This will be committed shortly.

Here's what example Imagenet data paths should look like, starting at the root DeOldify project folder. You may or may  not have the exact images I show here but the point is to just illustrate structure.  

training image x:  /data/imagenet/ILSVRC/Data/CLS-LOC/bandw/train/n01440764/n01440764_18.JPEG
training image y:  /data/imagenet/ILSVRC/Data/CLS-LOC/train/n01440764/n01440764_18.JPEG

validation image x:  /data/imagenet/ILSVRC/Data/CLS-LOC/bandw/val/n01440764/n01440764_18.JPEG
validation image y:  /data/imagenet/ILSVRC/Data/CLS-LOC/val/n01440764/n01440764_18.JPEG

**Comment by jantic** [comment]
Ok I've committed the update to ColorizeTrainingVideo.

**Comment by miltosc** [comment]
Thank you very much!
